beacon-chain/core/blocks/signature.go

```diff
@@ -204,12 +204,7 @@ func createAttestationSignatureBatch(
 			return nil, err
 		}
 		indices := ia.GetAttestingIndices()
-		pubkeys := make([][]byte, len(indices))
-		for i := 0; i < len(indices); i++ {
-			pubkeyAtIdx := beaconState.PubkeyAtIndex(primitives.ValidatorIndex(indices[i]))
-			pubkeys[i] = pubkeyAtIdx[:]
-		}
-		aggP, err := bls.AggregatePublicKeys(pubkeys)
+		aggP, err := beaconState.AggregatedKeyFromIndices(indices)
```

Comment:
Nit: The previous function is `AggregatePublicKeys`. Should'nt this function name `AggregateKeyFromIndices`?